bundles/admin/admin/DefaultViews.js

```diff
@@ -108,7 +107,7 @@ Oskari.clazz.define('Oskari.admin.bundle.admin.DefaultViews', function(locale, p
             type: 'POST',
             dataType: 'json',
             data: data,
-            url: sb.getAjaxUrl(),
+            url: Oskari.urls.getRoute(),
```

Comment:
Same here: give 'SystemViews' to getRoute() and take action_route: 'SystemViews' out of the payload.